setup.py

```diff
@@ -63,12 +63,15 @@
 ]
 
 install_requires = [
+    'dojson>=1.0.0',
```

Comment:
As ``DoJSONSerializer`` goes to invenio-marc21, then this one can probably be taken out